src/ucp/wireup/wireup.c

```diff
@@ -1413,71 +1473,76 @@ ucp_wireup_replace_wireup_msg_lane(ucp_ep_h ep, ucp_ep_config_key_t *key,
                                    uct_ep_h *new_uct_eps,
                                    const ucp_lane_index_t *reuse_lane_map)
 {
-    uct_ep_h uct_ep = NULL;
-    ucp_lane_index_t old_lane, new_wireup_lane;
-    ucp_wireup_ep_t *old_wireup_ep, *new_wireup_ep;
+    uct_ep_h uct_ep   = NULL;
+    int am_need_flush = ucp_wireup_is_am_need_flush(ep, reuse_lane_map);
```

Comment:
ucp_wireup_replace_wireup_msg_lane is called from ucp_wireup_check_config_intersect which already calculates ucp_wireup_is_am_need_flush, maybe we can pass it as a parameter?

Reply:
why is it more readable than calling the getter function in 2 places?